src/components/explore/ComparisonSummary.svelte

```diff
@@ -2,7 +2,6 @@
 import { format } from 'd3-format';
 import Tweenable from 'udgl/data-graphics/motion/Tweenable.svelte';
 
-let pFmt = format('.0%');
```

Comment:
After this is removed, we get:

`  2:10  error  'format' is defined but never used                               no-unused-vars`